support/cas-server-support-gauth-jpa/src/main/java/org/apereo/cas/adaptors/gauth/GoogleAuthenticatorJpaTokenRepository.java

```diff
@@ -36,7 +36,7 @@ public GoogleAuthenticatorJpaTokenRepository(final long expireTokensInSeconds) {
     @Override
     public void cleanInternal() {
         final int count = this.entityManager.createQuery("DELETE FROM " + GoogleAuthenticatorToken.class.getSimpleName()
-                + " r where r.issuedDateTime>= :expired")
+                + " r where r.issuedDateTime<= :expired")
```

Comment:
Could you explain this bit please?

Reply:
When I set the windowSize, for example to 7, and I use the last possible token, the registry is flushed before the token is really expired

Reply:
Could you add a few test cases to account for this scenario please, for all repository implementations?

Reply:
Sorry but I don't have enough time right now.
And I only use Google Auth in my tests.